napari/components/camera.py

```diff
@@ -0,0 +1,102 @@
+from ..utils.events import EmitterGroup
+
+
+class Camera:
```

Comment:
@sofroniewn you don't want to dataclass this up? ;)

Reply:
follow up PR!! but will be great